music_assistant/providers/plex/__init__.py

```diff
@@ -936,6 +936,15 @@ async def get_stream_details(self, item_id: str, media_type: MediaType) -> Strea
         media_part: PlexMediaPart = media.parts[0]
         audio_stream: PlexAudioStream = media_part.audioStreams()[0]
 
+        plex_data = {
```

Comment:
One possible small enhancement is to specify the dataformat with a TypedDict so you have a bit of help in the editor what to expect for the fields.